tdp/cli/utils.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+
+from tdp.core.dag import DAG_EXTENSION, DAG_FOLDER_NAME, Dag
+
+
+def create_dag_from_collection_path(collection_path):
```

Comment:
Why not put this function as a static method inside `Dag` class and name it `from_collection()` ?